manager/system_handler.py

```diff
@@ -148,6 +163,7 @@ def get_systems(self):
         data = []
         for system in system_view:
             record = {}
+            system['last_evaluation'] = system['last_evaluation'].isoformat() if system['last_evaluation'] else ''
```

Comment:
Is there any way for last_eval to be null/empty, if we start with SystemView being built with last_evaluation.is_null(False)?

Reply:
It's not, first, I was trying to return all systems including systems with null last_evaluation, then I added the is_null(False). Shall I remove the condition? But it doesn't harm anything.

Reply:
No worries - I wanted to make sure I was understanding the intent correctly.